List: update Selection guide (#4911)
DevExpress/devextreme-documentation: api-reference/10 UI Components/dxList/1 Configuration/selectByClick.md

Specifies whether an item is selected if a user clicks it.

---

> If you [enable selection](/api-reference/10%20UI%20Components/dxList/1%20Configuration/selectionMode.md '/Documentation/ApiReference/UI_Components/dxList/Configuration/#selectionMode') and [selection controls](/Documentation/ApiReference/UI_Components/dxList/Configuration/#showSelectionControls) in the List component, checkboxes appear near each item. When you set this property to `false`, users can select List items only by checking checkboxes. When you assign `true` to this property, users can click anywhere on an item to select it.

Reviewer: ```suggestion
If you [enable selection](/api-reference/10%20UI%20Components/dxList/1%20Configuration/selectionMode.md '/Documentation/ApiReference/UI_Components/dxList/Configuration/#selectionMode') and [selection controls](/Documentation/ApiReference/UI_Components/dxList/Configuration/#showSelectionControls) in the List component, checkboxes appear near each item. Assign `true` to this property to allow users to click anywhere on an item to select it, or `false` to select List items only by checking checkboxes. 
```
